subsys/bluetooth/controller/ll_sw/openisa/lll/lll.c

```diff
@@ -52,7 +52,7 @@ static struct {
 } event;
 
 /* Entropy device */
-static const struct device *dev_entropy;
+static const struct device *dev_entropy = DEVICE_DT_GET(DT_CHOSEN(zephyr_entropy));
```

Comment:
Should lack of an entropy device really be a build error here too?

Reply:
It looks like the module is not operational without this device, so I think so. Looks like `init_reset	` error code is propagated up to the BT HCI open() method. Maybe I'm wrong, will defer judgement to maintainers.

Reply:
@dleach02 can you please chime in here, or help us find the right person to make a call about this?